CMakeLists.txt

```diff
@@ -2,6 +2,8 @@ cmake_minimum_required()
 
 project(QuantLib)
 
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
Thanks!  Can this be overridden if someone wants to use a later version of the standard?

Reply:
Hm I don't know. Maybe change it to
```suggestion
if(NOT "${CMAKE_CXX_STANDARD}")
    set(CMAKE_CXX_STANDARD 11)
endif()
```